### PR TITLE
professions.json: Add 24 new professions, adjust old ones slightly

### PR DESCRIPTION
**Pull request by shimmerine**
#### Summary
Content "Add 24 new professions, adjust old ones slightly"

#### Purpose of change
more professions = good right? while i was there i fixed some nonsense related to charge removal.

#### Describe the solution
24 new professions roughly follow the following categories:

pros: Master Chef, Fashion Designer

criminals: Arsonist, Bomb Maker, Meth Cook (the one that isn't in jail), Contract Sniper,  Bootlegger, Sawbones, Illegal Gunsmith 

bank robbers: Trigger Man, Lookout, Getaway Driver

regular occupations: Pop Star, Bouncer, Chauffeur (Chauffeuse), Cage Fighter, Businessman, Food Critic, Sommelier, Carpenter, Exotic Dancer, Golfer

N/A:Stoner, Furry

in addition to new professions, some old ones have been tweaked slightly. charges have been increased for tailors kit, rebreather mask. change hillbilly's weapon to taurus raging judge loaded with 410 shells (because thats the embodiment a hillbilly gun for me), change some attire to be more appropriate for some professions. rebalance some skills and point costs very marginally.

#### Describe alternatives you've considered
not doing any of this!

#### Testing
started a new game with all new professions. no errors. no charge related errors in the character creation menu either.

#### Additional context
i added the furry profession, though i only became aware of #66488 as i was writing out the changes for this pr. generally speaking i think this is acceptable given my profession is slightly more granular, however our changes were eerily similar (we both chose wolf_suit for male and feline_suit for female). given the similarity of the content im willing to scrub it from my pr if requested.

### CODE REVIEW

**Comment by irisanuwu** [comment]
the solution to the charge problem came while i wasnt looking. resolving this

**Comment by mqrause** [comment]
To fix conflicts you need to rebase/merge master into your branch.

**Comment by I-am-Erk** [comment]
I'm not sure why your basic instinct seems to be to buff the stats on all the professions you've hit. Do you have a rationale for this?

**Comment by irisanuwu** [comment]
> I'm not sure why your basic instinct seems to be to buff the stats on all the professions you've hit. Do you have a rationale for this?

im trying to stick to the "<3 is beginner, 3-5 is intermediate, 6-7 is expert, 8-10 is master" philosophy.
i think it stands to reason that given that, say, a rifle hunter has been hunting "since they were a child" or that swat officers, generally speaking, possess an "expert" level of handling in real life that my changes are appropriate. i consider the metric above a pretty accurate one and it helps in matters of making professions more true to life. though, if you find them disagreeable i have no qualms to reverting them back the way they used to be.

**Review by kevingranade — CHANGES_REQUESTED** [review]
> more professions = good right?

No

> while i was there i fixed some nonsense related to charge removal.

separate PR please

**Comment by irisanuwu** [comment]
> > more professions = good right?
> 
> No
> 
> > while i was there i fixed some nonsense related to charge removal.
> 
> separate PR please

please point out the professions you want me to remove

File: data/json/professions.json [hunk 1]
```diff
@@ -6265,5 +6305,890 @@
...
+    "skills": [ { "level": 4, "name": "dodge" } ],
+    "flags": [ "NO_BONUS_ITEMS" ],
+    "traits": [ "DEFT" ],
+    "items": { "male": [ "bikini_bottom_short" ], "female": [ "bikini_top_short", "bikini_bottom_short", "heels" ] }
+  },
+  {
+    "type": "profession",
+    "id": "bank_robber",
+    "name": "Trigger Man",
+    "description": "The bank robbery was going exactly according to plan… until the brains of the operation got bit by the inside man.  To add insult to injury, your getaway driver decided to bail on you.  At least you don't have to split the haul now.",
```
Comment (TheShadowFerret) [inline data/json/professions.json:7085]: Where did the haul go in this prof's inventory 🤔

Comment (irisanuwu) [reply data/json/professions.json:7085]: { "item": "duffelbag", "contents-group": "bank_score" },

 {
    "type": "item_group",
    "subtype": "collection",
    "id": "bank_score",
    "entries": [
      { "item": "money_strap_hundred", "count": 20 },
      { "item": "money_strap_fifty", "count": 20 },
      { "item": "money_strap_twenty", "count": 20 }
    ]
  },

Comment (TheShadowFerret) [reply data/json/professions.json:7085]: I can't read :D

**Comment by irisanuwu** [comment]
slimmed down to 24. at first blush the changes i made seem compliant. draft until i get further confirmation.

File: data/json/professions.json [hunk 2]
```diff
@@ -2065,7 +2089,7 @@
     "id": "horse_cop",
     "name": "Mounted Police Officer",
     "requirement": "achievement_reach_police_station",
-    "description": "You have spent many long years forging a bond with your equine companion, from running down common thugs, to entertaining kids when you ride on by.  The cataclysm is upon you, but at least you have eachother.",
+    "description": "You have spent many long years forging a bond with your equine companion, from running down common thugs, to entertaining kids when you ride on by.  The Cataclysm is upon you, but at least you have eachother.",
```
Comment (I-am-Erk) [inline data/json/professions.json:2092]:
```suggestion
    "description": "You have spent many long years forging a bond with your equine companion, from running down common thugs, to entertaining kids when you ride on by.  The Cataclysm is upon you, but at least you have each other.",
```

File: data/json/professions.json [hunk 3]
```diff
@@ -6265,5 +6305,605 @@
...
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "panties", "bra" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "arsonist",
+    "name": "Arsonist",
```
Comment (I-am-Erk) [inline data/json/professions.json:6397]: I don't think arsonist is really a profession. More of a background.

File: data/json/professions.json [hunk 4]
```diff
@@ -6265,5 +6305,605 @@
...
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "panties", "bra" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "bomber",
+    "name": "Bomb Maker",
```
Comment (I-am-Erk) [inline data/json/professions.json:6427]: Again, this doesn't seem like a profession to me. Have you met many professional bomb makers?

File: data/json/professions.json [hunk 5]
```diff
@@ -6265,5 +6305,605 @@
...
+      "female": [ "panties", "bra" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "meth_cook",
+    "name": "Meth Cook",
+    "description": "You were a chemical cook, making a small fortune from selling your product in bulk.  The Cataclysm may have left you without a job, but you're certain you can put your skills to good use.",
+    "points": 3,
+    "skills": [ { "level": 6, "name": "chemistry" }, { "level": 3, "name": "firstaid" } ],
```
Comment (I-am-Erk) [inline data/json/professions.json:6457]: A professional meth cook probably has some self defense skills.

File: data/json/professions.json [hunk 6]
```diff
@@ -6265,5 +6305,605 @@
...
+        "entries": [ { "group": "charged_smart_phone" }, { "group": "charged_laptop" } ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "panties", "bra" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "bootlegger",
+    "name": "Bootlegger",
```
Comment (I-am-Erk) [inline data/json/professions.json:6602]: While people do probably bootleg liquor professionally, it seems much more like a hobby.

File: data/json/professions.json [hunk 7]
```diff
@@ -6265,5 +6305,605 @@
...
+        ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "bra", "panties" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "idol",
+    "name": "Pop Star",
```
Comment (I-am-Erk) [inline data/json/professions.json:6729]: I wouldn't add something like this unless you first want to go through and add some npc reactions to you being famous. Which would be pretty awesome and fun, but should be a PR of its own.

File: data/json/professions.json [hunk 8]
```diff
@@ -6265,5 +6305,605 @@
...
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "bra", "panties" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "golfer",
+    "name": "Golfer",
```
Comment (I-am-Erk) [inline data/json/professions.json:6895]: Again, hobby not profession

**Review by I-am-Erk — COMMENTED** [review]
I haven't reviewed the balance in everything, just the appropriateness.

Side note, I love the idea of a pop star start that changes the initial reactions if several npcs, if someone wants to do that.

File: data/json/professions.json (hunk 8 above)
Comment (sadenar) [reply data/json/professions.json:6895]: there's such a thing as professional golfers

Comment (I-am-Erk) [reply data/json/professions.json:6895]: > "description": "You were a golf enthusiast, playing with your friends on weekends.  Let's hope you practiced your swing.",

File: data/json/professions.json [hunk 9]
```diff
@@ -628,7 +650,7 @@
     "name": "Video Blogger",
     "description": "You made a career out of creating and commentating video content and posting it on various platforms.  Now most of your fans are dead, alongside your equipment.  Maybe you didn't cultivate the most practical skillset through your career, but you have learned a thing or two about troubleshooting software and getting people to listen to you.",
     "points": 2,
-    "skills": [ { "level": 4, "name": "speech" }, { "level": 4, "name": "computer" } ],
+    "skills": [ { "level": 4, "name": "computer" }, { "level": 4, "name": "speech" } ],
```
Comment (kevingranade) [inline data/json/professions.json:653]: What are you doing here? Do not make non-functional changes.

**Review by kevingranade — CHANGES_REQUESTED** [review]
Drop the random balance changes, I've spotted several that are not obviously correct, and obviously correct tweaks are the only off-topic changes that should be bundled together in a PR.

File: data/json/professions.json [hunk 10]
```diff
@@ -6265,5 +6305,605 @@
       "male": [ "briefs" ],
       "female": [ "panties", "bra" ]
     }
+  },
+  {
+    "type": "profession",
```
Comment (kevingranade) [inline data/json/professions.json:6310]: Drop it, too "tailored" for what a player wants.

File: data/json/professions.json [hunk 11]
```diff
@@ -6265,5 +6305,605 @@
...
+        "items": [ "suit", "dress_shoes", "socks" ],
+        "entries": [ { "group": "charged_smart_phone" }, { "item": "wine_noir", "custom-flags": [ "auto_wield" ] } ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "panties", "bra" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "contract_sniper",
```
Comment (kevingranade) [inline data/json/professions.json:6503]: nope

File: data/json/professions.json [hunk 12]
```diff
@@ -6265,5 +6305,605 @@
...
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "panties", "bra" ]
+    },
+    "missions": [ "MISSION_ASSASSINATION" ]
+  },
+  {
+    "type": "profession",
+    "id": "carpenter",
```
Comment (kevingranade) [inline data/json/professions.json:6547]: This doesn't really provide anything distinctive in addition to the crafting hobby, part of the point of hobbies is to reduce clutter for skillset-specific professions.

File: data/json/professions.json [hunk 13]
```diff
@@ -6265,5 +6305,605 @@
...
+          { "item": "nail", "charges": 50 }
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "bra", "panties" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "food_critic",
```
Comment (kevingranade) [inline data/json/professions.json:6584]: no point?

File: data/json/professions.json [hunk 14]
```diff
@@ -6265,5 +6305,605 @@
...
+          { "item": "moonshine", "custom-flags": [ "auto_wield" ] }
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "panties", "bra" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "businessman",
```
Comment (kevingranade) [inline data/json/professions.json:6623]: redundant with other professions

File: data/json/professions.json [hunk 15]
```diff
@@ -6265,5 +6305,605 @@
...
+    }
+  },
+  {
+    "type": "profession",
+    "id": "chauffeur",
+    "name": { "male": "Chauffeur", "female": "Chauffeuse" },
+    "description": "You were the personal driver of an A-list celebrity.  When shit hit the fan, you decided to speed off instead of waiting for them.  Now it's just you and your limo.",
+    "points": 3,
+    "skills": [ { "level": 6, "name": "driving" }, { "level": 3, "name": "speech" } ],
+    "vehicle": "limousine",
```
Comment (kevingranade) [inline data/json/professions.json:6690]: nope

File: data/json/professions.json [hunk 16]
```diff
@@ -6265,5 +6305,605 @@
...
+    "flags": [ "NO_BONUS_ITEMS" ],
+    "items": {
+      "both": { "items": [ "socks", "tank_top" ] },
+      "male": [ "wolfsuit", "briefs" ],
+      "female": [ "felinesuit", "panties" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "stoner",
```
Comment (kevingranade) [inline data/json/professions.json:6760]: redundant

File: data/json/professions.json [hunk 17]
```diff
@@ -6265,5 +6305,605 @@
...
+          { "group": "charged_smart_phone" }
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "boy_shorts" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "illegal_gunsmith",
```
Comment (kevingranade) [inline data/json/professions.json:6781]: redundant

**Comment by kevingranade** [comment]
Other than Erk's point about being some kind of well-known personality being a neat interaction, I'm not seeing any of these that are distinctive enough, and a lot of them are completely redundant with existing professions and hobbies.

> more professions = good right? 

I think this is the crux of the issue, just adding more variations is not valuable.
We want to explore the set of options that fit in the game, but only ones that are meaningfully different. A profession with similar skills and equipment and a different description is just noise when creating a character.

**Comment by irisanuwu** [comment]
ok i guess ill just scrap this. thanks for taking a look

File: data/json/professions.json (hunk 12 above)
Comment (I-am-Erk) [reply data/json/professions.json:6547]: I disagree on this one, but it should maybe have a few more associated proficiencies first

**Comment by I-am-Erk** [comment]
I'd encourage you to consider making a couple of these into hobbies if they aren't already there, or seeing about making the pop star start because that would be extremely fun.

**Pull request closed without merging**